Library/ViewModels/FeatureFlagToolsViewModel.swift

```diff
@@ -47,12 +47,13 @@ public final class FeatureFlagToolsViewModel: FeatureFlagToolsViewModelType, Fea
       return features
     }
 
-    self.reloadWithData = features
+    let sortedFeatures = features
       .map { features in features.sorted { $0.feature.description < $1.feature.description } }
 
-    self.updateConfigWithFeatures = features
```

Comment:
You can see that previously `reloadWithData` would get sorted array while `updateConfigWithFeatures` would not ... this should be fixed now